eng/common/pipelines/templates/steps/get-code-owners.yml

```diff
@@ -0,0 +1,20 @@
+parameters: 
+  TargetUserVariable: ''
+  ServiceDirectory: ''
+  RetrieveCodeOwnersToolVersion: '1.0.0-dev.20211108.8'
+  DevOpsFeed: "https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json"
+steps:
+  - pwsh: |
+      Write-Host "$(RetrieveCodeOwnerToolVersion)"
+      Write-Host "$(RetrieveCodeOwnerToolName)"
+      dotnet tool install --tool-path . --add-source "${{ parameters.DevOpsFeed }}" --version "${{parameters.RetrieveCodeOwnersToolVersion}}" "Azure.Sdk.Tools.RetrieveCodeOwners"
+    displayName: 'Install tool RetrieveCodeOwners'
+    workingDirectory: '$(Agent.ToolsDirectory)'
+
+  - pwsh: |
+      ./retrieve-codeowners `
+        --target-directory "/sdk/${{ parameters.ServiceDirectory }}/" `
```

Comment:
Isn't the plan to move this into the update-docsms-metadat.yml so that you can call it once per package?